library/index.c

```diff
@@ -323,24 +323,25 @@ int eblob_index_blocks_fill(struct eblob_base_ctl *bctl)
 }
 
 
-static struct eblob_disk_control *eblob_find_on_disk(struct eblob_backend *b,
-		struct eblob_base_ctl *bctl, struct eblob_disk_control *dc,
+static int eblob_find_on_disk(struct eblob_backend *b,
```

Comment:
>read not the whole file but only a chunk containing range
Yes, this is what eblob_find_on_disk() actually does in this PR. I made a mistake saying that it reads whole file in my previous comment.